### ISSUE TITLE
Remind the author of overdue

### ISSUE THREAD

**Issue by nnmer** [f0y/due_date_reminder#27]
It could be nice to be able to choose in settings of the plugin whether to remind the author about overdue issues or not

**Comment by cforce** [comment]
Who is messaged at the moment? The assignee and not the author?

**Comment by cforce** [comment]
+1 for this

**Comment by nnmer** [comment]
Best of all, if it can be centralized checkbox, and checked by default. Each user will be able to uncheck it if they would like to disable notification to author

**Comment by f0y** [comment]
From my point of view an author is a person who is responsible for the issue. So he and not the user itself should decide whether he wants or does not  want receive the notifications. So its not that simple decision where to put that checkbox.

**Comment by nnmer** [comment]
okay, then let it be mandatory. This is very important part of the notifications. Author usually has a lot of opened issues, he for sure should receive a notice that some of the issues still not done! This makes the management more organized!

**Comment by nnmer** [comment]
looks that my comment 6 days before a little bit confusing. I mean that each users also can be an a author, and it could be nice if users will be able to switch on or off notifications to them if they are the author of some issues.
But as i said in a comment above, let it be mandatory, to notify the author about issues opened by him

**Comment by f0y** [comment]
Maybe the right way is to name that checkbox "remind me if I am an author" and left it checked by default. Is it what you mean?

**Comment by nnmer** [comment]
yes, yes, this will make life easier when i`m an author and i will be always updated if issue is overdue or close to due time
